src/librustc_target/spec/thumb_base.rs

```diff
@@ -50,6 +50,9 @@ pub fn opts() -> TargetOptions {
         // until we figure a way to add the pretty printers without requiring a volatile load cf.
         // rust-lang/rust#44993.
         emit_debug_gdb_scripts: false,
+        // LLVM is eager to trash the link register when calling `nounwind` functions, which
```

Comment:
Shouldn't this say `noreturn` rather than `nounwind`?

Reply:
Oof, of course I mess up this tiny change. Fixed.